pkg/transforms/transformer.go

```diff
@@ -356,6 +355,19 @@ func TransformRoutine(input chan *Event, output chan NodeEvent) {
 			}
 			trans = PolicyResourceBuilder(&typedResource)
 
+		case [2]string{"ConfigurationPolicy", "policy.open-cluster-management.io"}:
```

Comment:
(Optional) Looks like there might be a convention to use a variable for the API group similar to `APPS_OPEN_CLUSTER_MANAGEMENT_IO`
```suggestion
		case [2]string{"ConfigurationPolicy", POLICY_OPEN_CLUSTER_MANAGEMENT_IO}:
```